### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by qianzha**
I found that there is a missing key "ritual.bloodmagic.groundingRitual.groundingRange.info" at us_en.lang.

And these keys below are been setted twice with same value:
ritual.bloodmagic.condorRitual.info=Provides flight in an area around the ritual.
(at 679 line & 767 line)
ritual.bloodmagic.eternalSoulRitual.info=Capable of transferring Life Essence from a Network back into an Altar at a cost.
(at 680 line & 768 line)

### CODE REVIEW

**Comment by qianzha** [comment]
>  Got tired of making one of these primitive buildings to generate cobblestone? Got tired of having OSHA on you or accidentally extinguishing your lava source? 

(Introduction of Cobblestone Ritual at guide book)
What does OSHA mean here?

**Comment by keraldi** [comment]
OSHA is a US institution that regulates or enforces work safety laws.
https://en.wikipedia.org/wiki/Occupational_Safety_and_Health_Administration

In this context it's joking about lava sources being very dangerous and OSHA hunting the player down because lava is not safe to work with.

**Comment by qianzha** [comment]
> OSHA is a US institution that regulates or enforces work safety laws.
> https://en.wikipedia.org/wiki/Occupational_Safety_and_Health_Administration
> 
> In this context it's joking about lava sources being very dangerous and OSHA hunting the player down because lava is not safe to work with.

Got it,  thx!

**Comment by qianzha** [comment]
@3TUSK I just updated the zh_cn.lang of the guide book, which you did a large of translation of. Would you like to check it?

File: src/main/resources/assets/bloodmagicguide/lang/zh_CN.lang [hunk 1]
```diff
@@ -83,6 +100,23 @@ guide.bloodmagic.entry.ritual.timberman.info=种了一堆树？想必你需要
...
+guide.bloodmagic.entry.ritual.altarBuilder.info=##REQ-LORE##\n\t祭坛集结号是一个用于搭建新的祭坛金字塔的仪式，通过获取其连接容器中的方块来铺设（通常是主仪式石上的箱子）。\n\t激活消耗：450LP\n\t每次放置方块消耗：75LP
+guide.bloodmagic.entry.ritual.pump.info=##REQ-LORE##\n\t虹吸圣曲用于吸取仪式周围和下方的液体，存入主仪式石上方的适当容器（例如血之储罐）。吸取液体的位置将用石头填放，以防止流动的液体造成性能问题。\n\t激活消耗：500LP\n\t每次操作消耗：250LP
+guide.bloodmagic.entry.ritual.cobble.info=##REQ-LORE##\n\t厌倦了用原始的刷石机来产圆石？厌倦了被安监局找上门，或是不小心就灭掉你的熔岩源？这个仪式就是为你准备的！快速地产出难以匹敌的巨量圆石；用上高效又不过时的血液，而非危险又有限的熔岩（除非你是用血液产熔岩） \n\n\t/cut这个仪式默认下在上方生成圆石，便于获取与使用。 \n\t激活消耗：500LP\n\t每方块消耗：25LP
+guide.bloodmagic.entry.ritual.ellipsoid.info=##REQ-LORE##\n\t椭球的焦点能生成一个空心球体（默认下），球体材料来源于于主仪式石连接的容器（通常是主仪式石上方的箱子）。\n\t激活消耗：20,000LP\n\t每次放置方块消耗: 5LP
+guide.bloodmagic.entry.ritual.crystalHarvest.info=##REQ-LORE##\n\t碎晶裂纹能破坏其范围内所有恶魔意志晶簇的多余晶体，留下一块，并把破坏掉的晶体扔在地上。与“和风之唤”仪式结合，则可以把掉落的物品收集起来，储存进容器中便于后续使用。\n\t 激活消耗：40,000LP \n\t 每次操作消耗：50LP
+guide.bloodmagic.entry.ritual.forsakenSoul.info=##REQ-LORE##\n\t孤魂集会能加快仪式上方（默认下）的恶魔意志晶簇的生长。然而它并不能凭空生成意志，且晶簇的生长仍然需要恶魔结晶坛。运用机械师所识——生物具有“独特性”，要使晶簇更有效率的生长，就需要更多种类的生物死亡。同时，然而它也不能凭空生成恶魔意志灵域，所以你需要把意志晶体回馈于大气，或是将存有意志的地狱魂石放置于恶魔坩埚中。/cut这应该能免去外出收割恶魔意志的需要，除非供给于祭坛的生物不足且/或是把意志给消耗尽了。 \n\t激活消耗：40,000LP\n\t每伤害刻每生物消耗：2LP
```
Comment (3TUSK) [inline src/main/resources/assets/bloodmagicguide/lang/zh_CN.lang]: `运用机械师所识——生物具有“独特性”`
->
`运用生物”独特性“这一机制`

**Review by 3TUSK — CHANGES_REQUESTED** [review]
I was summoned.

File: src/main/resources/assets/bloodmagicguide/lang/zh_CN.lang (hunk 1 above)
Comment (qianzha) [reply src/main/resources/assets/bloodmagicguide/lang/zh_CN.lang]: Okay, it has been changed.

Comment (qianzha) [reply src/main/resources/assets/bloodmagicguide/lang/zh_CN.lang]: Are there any other problems?

**Review by 3TUSK — APPROVED** [review]

File: src/main/resources/assets/bloodmagicguide/lang/zh_CN.lang (hunk 1 above)
Comment (3TUSK) [reply src/main/resources/assets/bloodmagicguide/lang/zh_CN.lang]: Nope.

**Comment by qianzha** [comment]
@TehNut It's ready to be merged.

**Pull request merged**